### PR TITLE
fix: autoimport spend key + clarifications

### PR DESCRIPTION
**Pull request by jowparks**
## Summary
- Adds attempt to import raw spending key
- Clarifies the account import when a failure occurs
- swaps the order of the prompt for failed import to disambiguate the requested "name"
## Testing Plan

https://user-images.githubusercontent.com/26990067/217382192-ca672514-865d-4541-8404-8c81bce7fecf.mov

## Breaking Change

Is this a breaking change? If yes, add notes below on why this is breaking and
what additional work is required, if any.

```
[ ] Yes
```

### CODE REVIEW

File: ironfish-cli/src/commands/wallet/import.ts [hunk 1]
```diff
@@ -71,8 +51,10 @@ export class ImportCommand extends IronfishCommand {
     }
 
     if (account === null) {
-      this.log('No account to import provided')
-      return this.exit(1)
+      CliUx.ux.warn(
+        'Failed to decode the account from the provided input, please continue with the manual input below',
+      )
+      account = await this.manualImport()
```
Comment (NullSoldier) [inline ironfish-cli/src/commands/wallet/import.ts:57]: I think this flow got a bit worse in the failure case. If I enter an invalid key, I get asked twice, even though the validation already happens in importTTY. I don't think there's any reasonable way to get to this code path, right?

```
ironfish wallet:import
Paste the output of wallet:export: b4174e1c50453ac1638915136ab9eaff0e7e028c427754ef2163858eb5491d2
 ›   Warning: Failed to decode the account from the provided input, please continue with the manual input below
Enter the account spending key: b4174e1c50453ac1638915136ab9eaff0e7e028c427754ef2163858eb5491d2
Enter a new account name: jason2
InvalidPaymentAddress
```

Comment (jowparks) [reply ironfish-cli/src/commands/wallet/import.ts:57]: Yea your are right, we should just delete this path and exit if this occurs

Comment (jowparks) [reply ironfish-cli/src/commands/wallet/import.ts:57]: Screen Shot 2023-02-07 at 4 22 26 PM

Comment (jowparks) [reply ironfish-cli/src/commands/wallet/import.ts]: Screen Shot 2023-02-07 at 4 49 21 PM

**Review by NullSoldier — APPROVED** [review]

**Pull request merged**